packages/editor/src/lib/editor/managers/FontManager.ts

```diff
@@ -94,7 +94,11 @@ export class FontManager {
 				const shapeUtil = this.editor.getShapeUtil(shape)
 				return shapeUtil.getFontFaces(shape)
 			},
-			{ areResultsEqual: areArraysShallowEqual, areRecordsEqual: (a, b) => a.props === b.props }
+			{
+				areResultsEqual: areArraysShallowEqual,
+				// @ts-expect-error
+				areRecordsEqual: (a, b) => a.props.richText === b.props.richText,
```

Comment:
Not sure how best to handle this because the editor doesn't know which properties a shape uses for its fonts. I think it's best if we paint over this for now. At worst, this feature won't work unless the shape has a property named rich text.